backend/app.py

```diff
@@ -310,17 +338,42 @@ def get_user():
             "message": "User not found"
         }
     else:
-        return jsonify({
-            "success": True,
-            "message": "User found",
-            "data": {
-                "id": user.id,
-                "name": user.name,
-                "email": user.email,
-                "role": user.role,
-                "points_total": user.points_total
-            }
-        })
+        # check if this is a fellow inquiring about their point total,
+        # or if this is an admin inquiring about a fellow's total.
+        if user.role == "admin":
+            # get the specified info for admin
+            u_discord_id = request.args.get('name')
+            u_user = User.query.filter_by(name=u_discord_id).first()
+            if u_user is None:
+                return {
+                    "success": False,
+                    "message": "The requested fellow was not found."
+                }
+            else:
+                return jsonify({
+                    "success": True,
+                    "message": "Fellow found",
+                    "data": {
+                        "id": u_user.id,
+                        "name": u_user.name,
+                        "email": u_user.email,
+                        "role": u_user.role,
+                        "points_total": u_user.points_total
+                    }
```

Comment:
should we include the avatar link here instead of having a separate route?

Reply:
Yeah. would be great if you could send the link in the get_user route: `https://cdn.discordapp.com/avatars/{discord_id}/{avatar_hash}.png?size=128`

Reply:
Alright. I’ll refactor it into one endpoint!